### PR TITLE
USWDS - Settings: Updates docs  settings content

### PR DESCRIPTION
**Pull request by gscottqueen**
## USWDS - Settings: Updates docs  settings content

closes: Settings documentation refers to nonexistent directory #1143

## Description

Updates our documentation to set the reference to `dist/scss/theme`

### CODE REVIEW

**Comment by gscottqueen** [comment]
@mejiaj take a quick look and see if there is anything else you want me to tweak here

**Comment by mejiaj** [comment]
@gscottqueen we also need to update it later in on the paragraph

image

Last sentence
>You can use the styles.scss example file in the dist/src/stylesheets/theme directory as a starting point for your own Sass entry point:

File: pages/documentation/getting-started/settings.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ subnav:
   href: '#utilities-settings'
 ---
 
-A good way to get started with USWDS settings is to copy the `dist/src/stylesheets/theme` directory out of the `uswds` package and into the same directory where you keep your project Sass files. Then use the `@import` directive in Sass to import these settings **before importing the main USWDS Sass**. You can use the `styles.scss` example file in the `dist/src/stylesheets/theme` directory as a starting point for your own Sass entry point:
+A good way to get started with USWDS settings is to copy the `dist/scss/theme` directory out of the `uswds` package and into the same directory where you keep your project Sass files. Then use the `@import` directive in Sass to import these settings **before importing the main USWDS Sass**. You can use the `styles.scss` example file in the `dist/src/stylesheets/theme` directory as a starting point for your own Sass entry point:
```
Comment (mejiaj) [inline pages/documentation/getting-started/settings.md:22]: >example file in the `dist/src/stylesheets/theme`

Needs updating too.

Comment (gscottqueen) [reply pages/documentation/getting-started/settings.md]: ah ok, I see - got it

**Review by mejiaj — APPROVED** [review]

**Review by thisisdano — APPROVED** [review]

**Comment by thisisdano** [comment]
Ignoring unrelated Snyk fail

**Pull request merged**